VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -46,6 +46,10 @@
     public static final String EXTRA_DISCUSSION_COMMENT = "discussion_comment";
     public static final String EXTRA_DISCUSSION_TOPIC_OBJ = "discussion_topic_obj";
 
+    // FIXME: This would be injected, but dependency injector is not working right (causes runtime exception)
```

Comment:
This seems to be a result of the `@Inject` annotation on the `forceLogout()` method in this class, which causes the injector to encounter a circular dependency. That annotation is completely wrong and should be removed.